### PR TITLE
feature: add system design template

### PR DESCRIPTION
**Pull request by aretecode**
#### Live
https://app.excalidraw.com/l/1C24U3S2zX3/2OAzeG7zdku

#### Preview

<details><summary>Thumbnail</summary>

system-design-template

[live](https://app.excalidraw.com/l/1C24U3S2zX3/5s2ZSC2EiT4)

</details>

<details><summary>Annotated</summary>

system-design-analysis-and-flow--annotated

</details>

### CODE REVIEW

**Comment by ferminrp** [comment]
Hey @aretecode , this is missing the changes to the libraries file. Please, edit the libraries.json and add your entry to the end.

**Comment by aretecode** [comment]
@ferminrp updated.

File: libraries.json [hunk 1]
```diff
@@ -871,6 +871,20 @@
...
+    "authors": [
+      {
+        "name": "James Wiens",
+        "url": "https://github.com/aretecode"
+      }
+    ],
+    "source": "aretecode/system-design-template.excalidrawlib",
+    "preview": "aretecode/system-design-template.png",
+    "created": "2021-08-26",
+    "updated": "2021-09-13"
```
Comment (dwelle) [inline libraries.json:886]:
```suggestion
    "created": "2021-09-23",
    "updated": "2021-09-23"
```

**Comment by dwelle** [comment]
Thanks!

**Pull request merged**